modules/storage/variables.tf

```diff
@@ -99,3 +99,11 @@ variable "kms_key_name" {
   type        = string
   default     = null
 }
+
+variable "custom_placement_config" {
+  description = "Configuration of the bucket's custom location in a dual-region bucket setup. If the bucket is designated a single or multi-region, then the variable will be null."
+  type = object({
```

Comment:
@cgsyam thanks for the PR. Since there is only one variable `data_locations` in `custom_placement_config` should we just declare a list of string `data_locations` and use it in the code instead of using complicated object input?
Also also any change in this list will recreate the bucket. We should mention this in **BOLD** in README and also mention this in variable definition. Losing log bucket can be a major issue.

Reply:
Done

Reply:
Done